DBlock Game/DBlock Game/Program.cs

```diff
@@ -6,6 +6,11 @@ namespace DBlock_Game
 {
     class Program
     {
+        private int playerLocation;
```

Comment:
these variables should be defined in your method, not at the top

Reply:
open to suggestions on this one. i created at the top because i dont want them reset everytime my method is called